src/appmixer/utils/csv/AddRows/component.json

```diff
@@ -0,0 +1,96 @@
+{
+    "name": "appmixer.utils.csv.AddRows",
+    "author": "Harsha Surisetty <harsha@client.io>",
+    "description": "Append multiple rows to the end of a CSV file.",
+    "version": "1.0.0",
+    "dependencies": {
+        "module": "1.0.1"
+    },
+    "private": false,
+    "inPorts": [
+        {
+            "name": "in",
+            "schema": {
+                "properties": {
+                    "fileId": {
+                        "type": "string"
+                    },
+                    "delimiter": {
+                        "type": "string"
+                    },
+                    "rows": {
+                        "oneOf": [
+                            {
+                              "type": "array"
+                            },
+                            {
+                              "type": "string"
+                            }
+                          ]
+                    },
+                    "parseNumbers": {
+                        "type": "boolean"
+                    },
+                    "parseBooleans": {
+                        "type": "boolean"
+                    }
+                },
+                "required": [
+                    "fileId",
+                    "delimiter",
+                    "rows"
+                ]
+            },
+            "inspector": {
+                "inputs": {
+                    "fileId": {
+                        "type": "filepicker",
+                        "label": "File ID",
+                        "index": 1,
+                        "tooltip": "The ID of the CSV file."
+                    },
+                    "delimiter": {
+                        "type": "text",
+                        "label": "Delimiter",
+                        "index": 2,
+                        "defaultValue": ",",
+                        "tooltip": "A character to use as a delimiter between columns."
+                    },
+                    "rows": {
+                        "type": "text",
+                        "label": "Rows",
+                        "index": 3,
+                        "defaultValue": "[]",
+                        "tooltip": "The rows to append."
```

Comment:
it's not clear what format should we use. It has to be a JSON array so it would good to mention it, here, maybe with a simple example